addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -59,6 +57,12 @@
     --gutter-x: 30px;
 }
 
+.o_extra_menu_items .o_grid_mode {
+    display: flex !important;
+    row-gap: 0px !important;
+    column-gap: 0px !important;
```

Comment:
Note that in 16.0, there is no option to modify the gaps so they technically should be 0. But I still added them here in case someone changed them by using their own CSS rules or by modifying the html :slightly_smiling_face: